### PR TITLE
Add instructions in Contributing to not put issue # in commit msgs

### LINKED ISSUE #12140: Update the Contributing doc regarding commit message
We should probably update the contributing doc to read:

>Do not add the issue number in the PR title or commit message.

instead of its current:

>Do not add the issue number in the PR title

The document is located here: https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md under "Common Steps"

### PR DESCRIPTION
**Pull request by FlameFractal**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #12140 

#### Description

Added instructions in the CONTRIBUTING.md doc to not add issue number in commit messages.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ for further information
 4.  The title (also called the subject) of your PR should be descriptive of your
     changes and succinctly indicates what is being fixed.
 
-    -   **Do not add the issue number in the PR title**.
+    -   **Do not add the issue number in the PR title or commit message.**.
```
Comment (dhcodes) [inline CONTRIBUTING.md]: @FlameFractal please remove the extra period at the end of this line and then save the file, do a `git add [filename]`, then `git commit --amend`, then push the branch again with `-f`.

Thanks for catching the other spot to update this.

**Review by dhcodes — CHANGES_REQUESTED** [review]
Update the file to remove the period and amend per the instructions.

**Comment by FlameFractal** [comment]
@dhcodes Updated with the requested changes.

Just wanted to ask something basic. When I tried "git commit --ammend", it returned me an error "error: unknown option `ammend'"

I instead made a new commit and squashed it into previous one. But it would be very helpful if you could guide me as to what went wrong!

**Comment by raisedadead** [comment]
@FlameFractal you have a typo, it's not `ammend` its `amend`

> file, do a `git add [filename]`, then `git commit --amend`, then push the branch again with -f.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -319,7 +319,7 @@ nothing to commit, working directory clean
     add .` to add all unstaged files. Take care, though, because you can
     accidentally add files you don't want added. Review your `git status` first.
 
-6.  Commit your edits: `git commit -m "Brief Description of Commit"`
+6.  Commit your edits: `git commit -m "Brief Description of Commit"`. Do not add the issue number in the commit message.
```
Comment (raisedadead) [inline CONTRIBUTING.md]: Please remove this additional line that you have added in your PR.
> . Do not add the issue number in the commit message.

**Review by raisedadead — CHANGES_REQUESTED** [review]
Please remove the additional lines that you have added.

File: CONTRIBUTING.md (hunk 2 above)
Comment (FlameFractal) [inline CONTRIBUTING.md]: @raisedadead It would be very confusing if someone reads that instruction way later than where commits are explained. @dhcodes

Comment (dhcodes) [inline CONTRIBUTING.md]: @raisedadead yeah I'm actually not opposed to this addition. Your thoughts?

Comment (raisedadead) [inline CONTRIBUTING.md]: Hmm, I never thought of this, thanks @FlameFractal @dhcodes . Approved

**Review by raisedadead — APPROVED** [review]

**Review by dhcodes — APPROVED** [review]

**Comment by dhcodes** [comment]
#LGTM 

Thanks and congratulations on your first contribution to FreeCodeCamp! 👍 🌮 🎉 💯

**Comment by FlameFractal** [comment]
My first contribution to opensource actually. Thank you both @dhcodes @raisedadead 😃

**Comment by dhcodes** [comment]
Even better :)

**Comment by raisedadead** [comment]
### @FlameFractal Congratulations! Keep them coming. 🎉 ✨ 🎊

**Pull request merged**